.gitignore

```diff
@@ -11,3 +11,4 @@ src/test/resources/cloudant-2.properties
 *.iml
 build/
 .gradle/
+gradle.properties
```

Comment:
I thought it would reduce the risk of people committing credentials after running tests.